### ISSUE TITLE
Cannot open MenuItem on click

### ISSUE THREAD

**Issue by Fuzzy2319** [kikipoulet/SukiUI#52]
I cannot open my MenuItem "File" with my mouse but I can with a keyboard

Code:
```xml
<Window [...]
        Classes="NakedWindow Blurred">
    <suki:DesktopPage Title="OrigamiEditor"
                      MenuVisibility="True">
        <suki:DesktopPage.LogoContent>
            <Image Source="avares://OrigamiEditor/assets/logo.ico" MaxHeight="16"/>
        </suki:DesktopPage.LogoContent>
        <suki:DesktopPage.MenuItems>
                <MenuItem Header="File">
                    <MenuItem Header="Test" Foreground="{DynamicResource SukiText}" />
                    <MenuItem Header="Test" Foreground="{DynamicResource SukiText}" />
                    <MenuItem Header="Test" Foreground="{DynamicResource SukiText}" />
                    <MenuItem Header="Test" Foreground="{DynamicResource SukiText}" />
                </MenuItem>
                <MenuItem Header="Edit" Foreground="{DynamicResource SukiText}" />
                <MenuItem Header="Help" Foreground="{DynamicResource SukiText}" />
        </suki:DesktopPage.MenuItems>    
    </suki:DesktopPage>
</Window>
```

Actual behavior:
![image](https://github.com/kikipoulet/SukiUI/assets/61144247/b6403a03-2350-4130-92b4-f7d9e75993b7)

Expected behavior:
![image](https://github.com/kikipoulet/SukiUI/assets/61144247/b13298d8-15d2-405b-a744-2e3e31ace92e)

**Comment by kikipoulet** [comment]
Definitely not normal, the vertical alignment of the buttons on the right too. I guess it's the HitTest when adding Glassmorphism that became false.

I will fix that this week.

**Comment by kikipoulet** [comment]
Fixed in Nuget 5.2.2
